UPI Payments India/README.md

```diff
@@ -0,0 +1,46 @@
+<div align = 'center'>
+  <h1>UPI Payments India</h1>
+  </div>
+  
+<img src = "https://upload.wikimedia.org/wikipedia/commons/e/e1/UPI-Logo-vector.svg">
+
+### Goal
+The goal of this project is to build a machine learning model with highest accuracy, which will predict the UPI Payments trend.
+
+### Dataset
+The dataset for this project is taken from the Kaggle dataset website. Here is the link for the dataset, https://www.kaggle.com/bhatnagardaksh/upipaymentsindia
+
+### What Have I Done?
+- Imported all the required libraries and dataset for this project.
+- Exploratory Data Analysis and Visualizing different aspects of the dataset.
+- Finding number of observations and outliers in the dataset.
+- Plotting different attributes of the dataset.
+- Applying different Machine Learning algorithms for the Model Creation.
+  - Split the dataset into 80:20 ratio for training and testing purpose.
+  - Logistic Regression deployment.
+  - Decision Tree Classifier Algorithm deployment.
+  - Random Forest Classifier Algorithm deployment.
+  - Linear Regression Algorithm Deployment
+- Create the Model using the best fitted algorithm.
+
+### Libraries used:
+1. numpy.
+2. pandas.
+3. matplotlib.
+4. seaborn.
+5. sklearn.
+
+### Visualization and EDA of different attributes:
+"<img width="538" alt="Screenshot 2022-07-15 at 11 59 40 AM" src="https://user-images.githubusercontent.com/77090462/179170493-86eaee49-90c7-434e-9153-a99b99155968.png">
+
+### Model Creation:
+Here the Heatmap or matrix creation the Model.
+
+"<img width="707" alt="Screenshot 2022-07-15 at 12 00 20 PM" src="https://user-images.githubusercontent.com/77090462/179170711-4874f6e0-b691-4a67-b861-0f286d3fe59e.png">
+
+
+Hence, Decision tree and Random Forest Algorithms are used for creating the model.
```

Comment:
Add a section as "Conclusion", and conclude that, which models are the best fitted for this dataset with the accuracy checking parameter score. This will bring more precise check for the models.